packages/lesswrong/server/callbacks/sunshineCallbackUtils.tsx

```diff
@@ -1,11 +1,16 @@
 import Users from "../../lib/collections/users/collection";
 import { getCurrentContentCount } from '../../components/sunshineDashboard/SunshineNewUsersInfo';
+import { Comments } from "../../lib/collections/comments";
+import { ModeratorActions } from "../../lib/collections/moderatorActions";
+import { createMutator } from "../vulcan-lib";
+import { COMMENT_LOW_QUALITY_WARNING, COMMENT_MEDIOCRE_QUALITY_WARNING, isActionActive } from "../../lib/collections/moderatorActions/schema";
+import { forumTypeSetting } from "../../lib/instanceSettings";
 
 /** This function contains all logic for determining whether a given user needs review in the moderation sidebar.
  * 
  * It's important this this only be be added to async callbacks on posts and comments, so that postCount and commentCount have time to update first
  */
-export async function triggerReviewIfNeeded(userId: string) {
+export async function triggerReviewIfNeeded(userId: string, newModeAction?: true) {
```

Comment:
@Raemon I don't like this variable name. Forces me to look inside the function to know what it does. Seems it's function is something like "override", in which case call it that.

(Tagging you 'cause Robert says you thought it should be this way.)